toolchain/parse/node_kind.def

```diff
@@ -286,12 +286,14 @@ CARBON_PARSE_NODE_KIND(StructComma)
 CARBON_PARSE_NODE_KIND(StructLiteral)
 CARBON_PARSE_NODE_KIND(StructTypeLiteral)
 
+CARBON_PARSE_NODE_KIND(ExternLibraryModifier)
+CARBON_PARSE_NODE_KIND(ExternTokenModifier)
```

Comment:
Optional: this name makes it sound like it's a modifier for tokens, rather than a modifier that is a token. How about something like `BareExternModifier`?

Reply:
I think "token" is consistent with the name used below, `CARBON_PARSE_NODE_KIND_TOKEN_MODIFIER`

Reply:
Going a different direction, renaming to "ExternModifier" + "ExternModifierWithLibrary" since the former keeps the same name.

Reply:
LGTM